wasmcloud_host/README.md

```diff
@@ -1,21 +1,146 @@
 # wasmCloud Host - Web UI Dashboard
+
 This is the web UI dashboard that provides for a basic way to interact with a host and its associated lattice. This web application automatically starts the [host_core](../host_core/README.md) application as a dependency.
 
-## Starting the Host and Web UI Dashboard
+To learn more about wasmCloud, please view the [Documentation](https://wasmcloud.dev).
+
+
+## Building the host
+
+Clone the repo `wasmCloud/wasmcloud-otp` into some suitable development directory
+
+```console
+git clone https://github.com/wasmCloud/wasmcloud-otp.git
+```
+
+
+### Language Prerequisites
+
+Install the latest version of [Rust](https://www.rust-lang.org/tools/install) and [Elixir](https://elixir-lang.org/install.html).
+Elixir requires the Erlang VM to be installed, but many of the installation methods listed will take care of this for you.
+
+If building the `httpserver` provider from source, you'll also need [Go](https://golang.org/doc/install)
+
+### Environment Prerequisites
+
+1. Since the build of `wasmcloud-otp` will fail on version `16.xx.yy` of NodeJs, you should either install, or switch to, version `14.xx.yy` of NodeJS.
+
+    Typically, a NodeJS version management tool such as `nvm` or `fnm` is used.  For installation details, refer to the documentation for either [Fast Node Manager](https://github.com/Schniz/fnm) or [Node Version Manager](https://github.com/nvm-sh/nvm) (Mac/Linux only)
+
+    Once the Node management tool of your choice has been installed, install the latest Node 14 version, then switch to that version.  E.G.:
+
+    ```console
+    fnm install 14
+    fnm use 14
+    ```
+
+1. Ensure that your Node management tool has the correct shell environment variables.
+
+    In your shell initialisation script (such as `~/.bashrc` or `~/.profile`) add a line suitable for your Node managerment tool
+
+    E.G. if using Fast Node Manager `fnm`, add the line
+
+    ```shell
+    eval "$(fnm env)"
+    ```
+
+    Continue working in a new terminal window to ensure that the above environment variables are present.
+
+### NATS Messaging Server
+
+1. Install `nats-server`
+
+    ```console
+    brew install nats-server
+    ```
+
+1. Start the NATS server using the JetStream option
+
+    ```console
+    nats-server -js
+    ```
+
+    This terminal window is now occupied with the `nats-server` console logs
+
+### Local Server Build
+
+In a new terminal, change into the `wasmcloud-otp` folder, then change into the `assets` folder underneath `wasmcloud_host`
+
+1. `cd wasmCloud/wasmcloud-otp/wasmcloud_host/assets`
 
-To start the wasmCloud host and web ui, cd to this folder (wasmcloud_host), and type or paste these commands:
+1. Run `npm install`.
+
+    As long as you have switched to version `14.xx.yy` of NodeJs, this should run successfully
+
+1. Change up one directory level and run `make run-interactive`.  This will build the wasmCloud server and connect it to your already-started NATS server
```

Comment:
you can also `make run` to start the server, and `make build` to compile it. I would maybe reference the `wasmcloud_host` readme (https://github.com/wasmCloud/wasmcloud-otp/blob/main/host_core/README.md)